### PR TITLE
Issues Table Added

### PR DESCRIPTION
**Pull request by haider-sheikh**
issue table added

### CODE REVIEW

File: app/models/issue.rb [hunk 1]
```diff
@@ -30,4 +30,5 @@ def transition_callback
     puts "Current event: #{aasm.current_event}"
     puts "Transition Callback: from #{aasm.from_state} to #{aasm.to_state}"
   end
+
```
Comment (hafsabatool7) [inline app/models/issue.rb]: Remove

File: app/views/home/index.html.erb [hunk 2]
```diff
@@ -111,6 +111,7 @@
           <div class="card-header py-3">
             <h6 class="m-0 font-weight-bold text-info"><%= t('home.issue_priority') %></h6>
           </div>
+
```
Comment (hafsabatool7) [inline app/views/home/index.html.erb]: Revert

File: app/views/projects/show.html.erb [hunk 3]
```diff
@@ -21,11 +21,47 @@
...
   <div class="col-lg-10 offset-lg-1">
     <div class = "my-custom-project-data-area">
-      <table class="table table-hover">
-      </table>
+      <% if @project_issues.any?%>
+        <br>
+        <table class="table">
+          <thead class="thead-dark">
+            <tr>
+              <th scope="col"><%= t('home.issue_title') %></th>
```
Comment (hafsabatool7) [inline app/views/projects/show.html.erb]: What is the purpose of scope="col" ?

File: app/views/projects/show.html.erb [hunk 4]
```diff
@@ -21,11 +21,47 @@
...
+              <th scope="col"><%= t('home.issue_type') %></th>
+              <th scope="col"><%= t('home.issue_priority') %></th>
+              <th scope="col"><%= t('home.issue_assignee') %></th>
+              <th scope="col"><%= t('home.issue_status') %></th>
+              <th scope="col" style="text-align: center;"><%= link_to t('projects.add_issue'),'#',class: 'btn my-primary-btn' %></th>
```
Comment (hafsabatool7) [inline app/views/projects/show.html.erb]: Avoid inline styling

File: config/locales/en.yml [hunk 5]
```diff
@@ -74,6 +74,11 @@ en:
     issue_priority: 'Issues Priorities'
     issue_priority_high: 'High'
     issue_priority_low: 'Low'
+    issue_title: 'Title'
+    issue_type: 'Type'
+    issue_priority: 'Priority'
+    issue_status: 'Status'
+    issue_assignee: 'Assignee'
```
Comment (hafsabatool7) [inline config/locales/en.yml]: Either move them under :issue key. Or make generalized labels like
`title: 'Title'`

File: app/views/teams/show.html.erb [hunk 6]
```diff
@@ -7,7 +7,7 @@
       </div>
     </div>
     <% if can? :new_user_for_team, @team %>
-      <%= link_to t('teams.add_user'), new_user_for_team_team_path,{class: 'btn my-primary-btn',remote: true, 'data-toggle' =>  "modal", 'data-target' => '#my_id_model_window'}  %>
+      <%= link_to t('teams.add_user'), new_user_for_team_team_path,{class: 'btn my-primary-btn',remote: true, 'data-toggle' =>  "modal", 'data-target' => '#my_id_model_add_member'}  %>
```
Comment (hafsabatool7) [inline app/views/teams/show.html.erb]: my_id_... ? Rename

File: app/views/projects/show.html.erb [hunk 7]
```diff
@@ -21,11 +21,47 @@
...
+                </td>
+              </tr>              
+            <% end %>
+          </tbody>          
+        </table>
+        <%== pagy_bootstrap_nav(@pagy) %>
+      <% else %>
+      <br>
+      <h6><%= t('projects.no_issue') %></h6>
+      <%= link_to t('projects.add_issue'),'#', class: 'btn my-primary-btn' %>
```
Comment (hafsabatool7) [inline app/views/projects/show.html.erb]: indent

File: app/assets/stylesheets/application.scss [hunk 8]
```diff
@@ -65,3 +65,6 @@ $pagination-link-btn-border-color: #ddd;
     }
   }
 }
+.table-center-buttons {
+  text-align: center;
```
Comment (hafsabatool7) [inline app/assets/stylesheets/application.scss]: Use bootstrap's existing classes if possible. For example, here bootstrap already provides text-center class so we don't need to declare our own

File: app/views/projects/show.html.erb [hunk 9]
```diff
@@ -35,7 +35,10 @@
               <th><%= t('issues.issue_priority') %></th>
               <th><%= t('issues.issue_assignee') %></th>
               <th><%= t('issues.issue_status') %></th>
-              <th style="text-align: center;"><%= link_to t('projects.add_issue'),'#',class: 'btn my-primary-btn' %></th>
+              <th class="table-center-buttons">
+                <% if can? :create, Issue %>
+                  <%= link_to t('projects.add_issue'),new_project_issue_path(@project),class: 'btn my-primary-btn' %></th>
```
Comment (hafsabatool7) [inline app/views/projects/show.html.erb:40]:
```suggestion
                  <%= link_to t('projects.add_issue'), new_project_issue_path(@project), class: 'btn my-primary-btn' %></th>
```

File: app/views/projects/show.html.erb [hunk 10]
```diff
@@ -44,12 +47,14 @@
...
-                <td><%= issue.assignee.name %></td>
+                <td><%= issue.assignee ? issue.assignee.name : t('issues.not_assigned') %></td>
                 <td><%= issue.status %></td>
-                <td style="text-align: center;">
-                  <%= link_to t('shared.delete'),'#',class: 'btn btn-sm btn-danger' %>
-                  <%= link_to t('shared.edit'),'#',class: 'btn btn-sm my-primary-btn ' %>
-                  <%= link_to t('shared.show'),'#',class: 'btn btn-sm my-primary-btn ' %>
+                <td class="table-center-buttons">
+                  <% if can? [:destroy, :update], issue %>
+                    <%= link_to t('shared.delete'), project_issue_path(issue.project, issue), method: :delete, data: { confirm: t('shared.deletion_confirmation_message', resource: Issue.model_name.human.downcase) }, class: 'btn btn-sm btn-danger' %>
```
Comment (hafsabatool7) [inline app/views/projects/show.html.erb]: `resource: Issue.model_name.human.downcase`
Why didn't we use label from yml here?

File: app/views/projects/show.html.erb [hunk 11]
```diff
@@ -44,12 +47,14 @@
                 <td><%= issue.title %></td>
                 <td><%= issue.issue_type %></td>
                 <td><%= issue.priority %></td>
-                <td><%= issue.assignee.name %></td>
+                <td><%= issue.assignee ? issue.assignee.name : t('issues.not_assigned') %></td>
```
Comment (hafsabatool7) [inline app/views/projects/show.html.erb]: issue&.assignee || t('issues.not_assigned')

Explore safe navigation operator **&.**

File: app/views/home/index.html.erb [hunk 12]
```diff
@@ -1,128 +0,0 @@
-<div class="d-flex flex-column">
```
Comment (hafsabatool7) [inline app/views/home/index.html.erb]: Why removed?

Comment (haider-sheikh) [reply app/views/home/index.html.erb]: because it is shifted to companies index page by Usman.

**Pull request merged**